template.hbs

```diff
@@ -133,9 +133,17 @@
           <td class='table-cells' colspan='4'>
             <img class='image' src='{{Certificate.Logo}}' />
           </td>
+          {{#if Certificate.Parties.GoodsReceiver}}
+          <td class='table-cells' colspan='6'></td>
+
+          <td class='table-cells' colspan='6'>
+            {{> company Certificate.Parties.Manufacturer }}
+          </td>
+          {{else}}
           <td class='table-cells' colspan='6'>
             {{> company Certificate.Parties.Manufacturer }}
           </td>
+          {{/if}}
```

Comment:
it would seem simpler to just do:

```html
    {{#if Certificate.Parties.GoodsReceiver}}
    <td class='table-cells' colspan='6'></td>
    {{/if}}
    <td class='table-cells' colspan='6'>
      {{> company Certificate.Parties.Manufacturer }}
    </td>
```

right ?